guides/release/components/helper-functions.md

```diff
@@ -315,7 +315,69 @@ components.
 ```
 
 In the component's template, you can then use the `person` object:
-
 ```handlebars {data-filename=app/components/greeting/template.hbs}
 Hello, {{@person.givenName}} {{@person.familyName}}
 ```
+
+### The `in-element` helper
+
+Using the [`{{in-element}}`](https://api.emberjs.com/ember/3.27/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) helper, you can render content into a specific DOM element that is in a different part of the page. For instance, we might want
+to render a modal, tooltip or dropdown.
+
+Let's say I wanted to show a dropdown when a button is clicked on.
+
+Below, we have the button we want to click on, a div element that will hold the dropdown and the dropdown component. We pass in `showDropdown` that will add the dropdown into our div.
+```handlebars {data-filename=app/components/some-component.hbs}
+  <button type="button" {{on "click" this.onClickShowDropdown}}>More Actions</button>
+  <div id="dropdown-destination" />
+
+  <MyDropdownComponent
+    @show={{this.showDropdown}}
+  />
+```
+
+When the user clicks on the button, the flag `showDropdown` will be set to `true`.
+```js {data-filename=app/components/some-component.js}
+  @tracked
+  showDropdown = false;
+
+  @action
+  onClickShowDropdown() {
+    this.showDropdown = true;
+  }
+```
+
+Since `showDropdown` is passed into `MyDropdownComponent` as `show`, when `show` is `true`, the `in-element` helper is activated. We pass in `destinationElement` into `in-element` which contains the destination DOM element. The `in-element` helper will take the content block and render into the destination element.
```

Comment:
I think this paragraph doesn't get to the point as well as it could. What do you think about:

```suggestion
The dropdown component uses the argument `@show` to activate the `in-element` helper. We must **provide the destination DOM element** to the helper. In other words, where should the helper render its block content?
```

?

Reply:
That's good